git-link.el

```diff
@@ -268,6 +268,13 @@ As an example, \"gitlab\" will match with both \"gitlab.com\" and
   :type '(alist :key-type string :value-type (group function))
   :group 'git-link)
 
+(defcustom git-link-extensions-rendered-raw '("org" "md" "rst" "adoc")
```

Comment:
We should add `markdown` and `asciidoc`. I know I've seen `.markdown` and I thought I'd seen `.asciidoc`; looking at [its Wikipedia entry](https://en.wikipedia.org/wiki/AsciiDoc) confirms this.

Reply:
Since in GitHub Land "raw" means [a content-type of `text/plain`](https://raw.githubusercontent.com/sshaw/git-link/master/README.md) we should use another word. "Plain" maybe? Open to other ideas.

Reply:
done :)